home/tests/test_models.py

```diff
@@ -1056,6 +1065,7 @@ def test_simple_template_submission(self) -> None:
         }
 
     # TODO: Find a better way to test quick replies
+    # Fritz -> I'm not sure what I meant here, nor do I have any ideas.  Will ask Rudi and/or Jeremy
```

Comment:
We're doing away with quick replies and replacing them with buttons, so this is perhaps not as necessary. (depending on how the buttons are tested)

Reply:
Agreed, removed